src/components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './calculator.css';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calculator">
+        <div className="output">0</div>
+        <div className="numbers">
+          <button type="button">7</button>
+          <button type="button">8</button>
+          <button type="button">9</button>
+          <button type="button">4</button>
+          <button type="button">5</button>
+          <button type="button">6</button>
+          <button type="button">1</button>
+          <button type="button">2</button>
+          <button type="button">3</button>
+          <button type="button" className="number0">0</button>
+          <button type="button">.</button>
+        </div>
+        <button type="button" className="operation">&#xf7;</button>
+        <button type="button" className="operation">x</button>
+        <button type="button" className="operation">-</button>
+        <button type="button" className="operation">+</button>
+        <button type="button" className="operation">=</button>
+        <button type="button" className="options">AC</button>
+        <button type="button" className="options">+/-</button>
+        <button type="button" className="options">%</button>
```

Comment:
- [ ] Everything is great the only small change I would suggest you is that please use the `font-weight` and background so that It matches the design provided by microverse.
### Your design
![his](https://user-images.githubusercontent.com/81727988/204499690-b228d0d9-8bf7-4963-9d1d-47207422ae1d.png)
### Design by Microverse
![figma](https://user-images.githubusercontent.com/81727988/204499766-8ea7e65e-51fd-4850-8602-3a88121bdbb7.png)

Reply:
@roshan-bajgain, thanks for the detailed review i have reverted as guided 
Patrick